### PR TITLE
Add generic `shapes` icon

### PR DESCRIPTION
**Pull request by danielbayley**
Represents `class`ification, a collection of some kind, or learning…

Related to #1055.

### CODE REVIEW

File: icons/shapes.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M15.6 10a.9.9 0 0 0 .9-.9c0-.2 0-.3-.1-.5l-3.6-6.2a.95.95 0 0 0-1.2-.3c-.1.1-.3.2-.3.3L7.6 8.7a.9.9 0 0 0 .8 1.3Z" />
+  <rect width="8" height="8" x="2" y="14" rx="2" />
+  <circle cx="18" cy="18" r="4" />
```
Comment (karsa-mistmere) [inline icons/shapes.svg:14]: I think maybe we should shrink this by 1 unit as such and better match similarly sized squares:

image


```suggestion
  <path d="M8.3 10a.7.7 0 0 1-.626-1.079L11.4 3a.7.7 0 0 1 1.198-.043L16.3 8.9a.7.7 0 0 1-.572 1.1Z" />
  <rect x="3" y="14" width="7" height="7" rx="1" />
  <circle cx="17.5" cy="17.5" r="3.5" />
```

Comment (karsa-mistmere) [reply icons/shapes.svg:14]: (And again, this version has actually been in my drafts for _ages_, do you work for Figma or something? 😆 )

Comment (danielbayley) [reply icons/shapes.svg]: > do you work for Figma or something? 😆

@karsa-mistmere If I did, I don’t think they’d be too happy to discover I exclusively use Illustrator 😅

**Comment by jguddas** [comment]
How does the triangle feel with 2px radius?

**Review by ericfennis — APPROVED** [review]
Nice one!

**Pull request merged**